File: pset1/greedy.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>



int main(void)
{
  float c;
  do
 {
  printf("O hai! How much change is owed?\n");
  c = get_float();
 }
  while (c < 0);

//multiply by 100 and round

int s = round (c * 100);

int quarters = s / 25 ;
int dimes = (s % 25) / 10 ;
int nickels = ((s % 25) % 10) / 5 ;
int pennies = (((s % 25) % 10) % 5);

int coins = quarters + dimes + nickels + pennies;
 printf ("%i\n", coins);
}

File: pset6/greedy.py
while True:
    print("O hai! How much change is owed?")
    h = float(input())
    if h > 0:
        break

change = round(h * 100)

q = change//25
d = (change % 25)// 10
n = ((change % 25) % 10) // 5
p = (((change % 25) % 10) % 5)

coins = q+ d + n + p
print("{}".format(coins))
